app/views/event_sessions/show.html.erb

```diff
@@ -75,10 +75,93 @@
         <%= link_to "", safe_url(@event_session.video_url), :class => 'fa-file-video-o fa fa-5x', :target => '_blank' %>
     </p>
     <% end %>
+    <div class="reaction">
```

Comment:
Can we move this entire change to a partial and include in `show.html.erb` plz? Just to ensure that comments related view code is in its own partial.